Others/rotate string/question.md

```diff
@@ -0,0 +1,26 @@
+796. Rotate String
+Solved
+Easy
+Topics
+Companies

```

Comment:
Hey Aarushi! These lines are extra, rest looks good to me :)

Reply:
made the changes please check @bh-g